less/main.less

```diff
@@ -0,0 +1,213 @@
+@mainColour : #fe5f55;
```

Comment:
Great job using a separate LESS file! You could even create other LESS files for your variables or for different sections